### ISSUE TITLE
OpenJDK source link is too general

### ISSUE THREAD

**Issue by mukrop** [crocs-muni/usable-cert-validation#111]
Clicking "source" for OpenJDK's documentation (section _Original error message_) only goes to general https://openjdk.java.net/ and not to the specific definition of the error.

**Comment by zacikpa** [comment]
Yes, this was intentional, since each error message is defined in a different place in the source code (thrown as an exception string). I will try to think of some solution, the issue can be left open.
